src/delfino/commands/typecheck.py

```diff
@@ -1,31 +1,20 @@
 """Type checking on source code."""
 
+from itertools import groupby
+from pathlib import Path
+from typing import List, Tuple
+
 import click
 
+from delfino.click_utils.filepaths import filepaths_argument
```

Comment:
I think this is a valid failure because this module doesn't exist?

Reply:
Thanks. I forgot to add newly created files to my previ commit.